README.md

```diff
@@ -34,16 +34,130 @@ that the operator is able to understand and process.
 * [git][git_tool]
 * [go] version 1.12.5+
 * [kubernetes] version v1.11.0+
-* [oc] version v3.11+
+* [kubectl] version v1.11+
```

Comment:
I'd directly put 1.13+ as minimum version.
In that way we make sure we follow current OpenShift release (4.1) and we are not tied to having to support older versions.
It's interesting to have 1.13+ as 1.11 did not support schema changes, did not have webhook conversions etc.

Reply:
Same applies for kubernetes version

Reply:
:+1:

Reply:
1.11 does support schema changes, but in a limited way. Once you update your CRD, older apiversions have to be update as well.

Reply:
Even if you "can do that" I don't think you want to have to change the original schema if you increase the APIversion. What's the point otherwise? In that case you would be changing the original version and that's it.
Also, no ability to perform incompatible changes, which is one of the other reasons to increase APIVersion

Reply:
It means you cannot have two apiversions active at a given point of time. But you can upgrade.

Breaking compatibility of previous  CRD versions has shown to be quite an issue. You need to provide schema conversion webhook and make sure no other API is depending on your API. You need a very good reason to do so, even in 1.13

Reply:
> It means you cannot have two apiversions active at a given point of time. But you can upgrade.

What do you mean with this? You can be serving several APIversions at the same time in Kubernetes for the same CRD. I'm not sure I'm understanding what you say

> Breaking compatibility of previous CRD versions has shown to be quite an issue. You need to provide schema conversion webhook and make sure no other API is depending on your API. You need a very good reason to do so, even in 1.13

That is not an argument against requiring 1.13+, just at the contrary, in case we do that (we should not do it lightly but we should be able to do it  ) we need 1.13+ : )

Reply:
`All specified versions must use the same schema`

Reply:
That does not mean you cannot have two apiversions active at the same time. It says that all apiversions should have the same schema

Reply:
ok, fair enough, you are right. to be specific you cannot have two schemas active but you can have to apiversion active :man_facepalming: 

but YOU CAN UPGRADE not breaking comp.

So, supporting 1.11 should not be discarded. Let's product decide, for now we keep open to all possibilities

Reply:
We can discuss it with product of course, but to me there are clearly technical specific characteristics that should be totally our responsability and that we should be responsible for them.

I think an operator should be able to perform incompatible changes at schema level and it should be able to have APIversions with different schemas. Kubernetes itself has that in their native objects, and so do several other operators. We have also found evidence that we might want to do that with 3scale-operator and we should be able to adapt to possible new changes in this case too that require that.